### ISSUE TITLE
General question on these labs

### ISSUE THREAD

**Issue by lordBrightLord** [MicrosoftLearning/PL-200-Power-Platform-Functional-Consultant#252]
Are these labs directly related and linked to the PL-200 Microsoft learns content? I've been going through the modules and cannot seem to find a direct link to land here.

**Comment by julians1966** [comment]
No, these labs, like most GitHub labs under Microsoft Learning, are designed for use with the Instructor Led Training (ILT) course for PL-200. Some GitHub labs are reused in Learn and vice versa .g. PL-300 and AI-900. 

These labs often require a preconfigured environment and software, you can go through them yourself, but you will be responsible for the setup and licenses.
